routes/web.php

```diff
@@ -43,9 +43,7 @@
 
         Route::resource('orders.items', 'OrderItemsController', ['only' => ['update']]);
 
-        Route::get('/', function () {
-            return ujs_redirect(route('admin.store.orders.index'));
-        });
+        Route::get('/', '\App\Http\Controllers\RedirectController')->name('redirect.admin.store.orders.index');
```

Comment:
well... shoot, I know this isn't supposed to happen, but "on my machine" it's just always dots. might be an issue on my end, let me try it again real quick